packages/api-derive/src/tx/signingInfo.ts

```diff
@@ -63,6 +66,20 @@ function signingHeader (api: DeriveApi): Observable<Header> {
   );
 }
 
+interface Aura {
+  slotDuration: u64 & AugmentedConsts<'rxjs'>;
+}
+
+function babeOrAuraPeriod(api: DeriveApi) : AnyNumber|undefined {
```

Comment:
```suggestion
function babeOrAuraPeriod(api: DeriveApi) : BN | undefined {
```

This will also require the following

```typescript
import type { BN } from '@polkadot/util';
```